src/install.js

```diff
@@ -49,6 +50,65 @@ questions.optional = [
 	},
 ];
 
+function checkSetupEnv() {
+	winston.info("checking env vars for setup info...");
+	let envNbbRe = /NBB_(?!DB_).*/
+	let envNbbDbRe = /NBB_DB_.*/
+	let envopts = {
+		"NBB_URL": "url",
+		"NBB_PORT": "port",
+		"NBB_SECRET": "secret", // only a "cookie" for web setup???
```

Comment:
It is used to sign the cookie. It's still used, but it's probably better to not have this option set in `config.json`. OK for this PR